### PR TITLE
fix: multireslyrics adapt to 0.9.20

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: multires-lyrics-search/README.md [hunk 1]
```diff
@@ -64,14 +64,14 @@ To make it easier for the user, we have built and published the [Docker image](h
 You can retrieve the docker image using:
 
 ```bash
-docker pull jinahub/app.example.multireslyricssearch:0.0.1-0.9.17
+docker pull jinahub/app.example.multireslyricssearch:0.0.2-0.9.17
```
Comment (cristianmtr) [inline multires-lyrics-search/README.md:67]:
```suggestion
docker pull jinahub/app.example.multireslyricssearch:0.0.2-0.9.20
```

File: multires-lyrics-search/README.md [hunk 2]
```diff
@@ -64,14 +64,14 @@ To make it easier for the user, we have built and published the [Docker image](h
...
 ```
 So you can pull from its latest tags.
 
 Then you can run it, and you can proceed to see the results in the browser as explained before
 
 ```bash
-docker run -p 65481:65481 jinahub/app.example.multireslyricssearch:0.0.1-0.9.17
+docker run -p 65481:65481 jinahub/app.example.multireslyricssearch:0.0.2-0.9.17
```
Comment (cristianmtr) [inline multires-lyrics-search/README.md:74]:
```suggestion
docker run -p 65481:65481 jinahub/app.example.multireslyricssearch:0.0.2-0.9.20
```

File: multires-lyrics-search/flows/index.yml [hunk 3]
```diff
@@ -8,16 +8,17 @@ pods:
   - name: encoder
     show_exc_info: true
     uses: pods/encode.yml
-    parallel: $JINA_PARALLEL
+    polling: any
+    shards: $JINA_PARALLEL
```
Comment (cristianmtr) [inline multires-lyrics-search/flows/index.yml:12]: use `JINA_SHARDS` naming

**Review by cristianmtr — CHANGES_REQUESTED** [review]
some minor things

File: multires-lyrics-search/flows/index.yml (hunk 3 above)
Comment (JoanFM) [reply multires-lyrics-search/flows/index.yml:12]: same reason as in cxmodal, next iteration, I have it only on the `os env` part and I would like to keep the code in sync with docker image

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**